service/ephemeral.go

```diff
@@ -59,11 +57,11 @@ func parseSize(size string) (int64, error) {
 func (s *service) ephemeralNodePublish(
 	ctx context.Context,
 	req *csi.NodePublishVolumeRequest) (
-	*csi.NodePublishVolumeResponse, error) {
-
+	*csi.NodePublishVolumeResponse, error,
+) {
 	if _, err := os.Stat(ephemeralStagingMountPath); os.IsNotExist(err) {
 		Log.Debug("path does not exist, will attempt to create it")
-		err = os.MkdirAll(ephemeralStagingMountPath, 0750)
+		err = os.MkdirAll(ephemeralStagingMountPath, 0o750)
```

Comment:
Just wondering, why do you need to get rid of canonic `0750` in favor of `0o750`? Is there any real benefit of it? Better code style or something?